test/binance-october-2022/Attacker.t.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../../src/binance-october-2022/CraftPayload.sol";
+
+contract BinanceAttackerTest is Test {
+    using stdStorage for StdStorage;
+
+    CraftPayload public craftPayload;
+
+    address inputAddress = 0xA9462F1880d0671aC9Cb7c3b411C484cdc624E94; // random Address
```

Comment:
use `https://book.getfoundry.sh/reference/forge-std/make-addr` instead of hardcoded address for random address.

Reply:
Good Idea 👍